Translations/de.lproj/loveView.strings

```diff
@@ -1,24 +1,24 @@
 ﻿
 /* Class = "NSTextFieldCell"; title = "Privacy Policy"; ObjectID = "272-yr-FLe"; */
-"272-yr-FLe.title" = "Privacy Policy";
+"272-yr-FLe.title" = "Datenschutzerklärung";
```

Comment:
In Google Translator this has an accepted translation: **Datenschutz-Bestimmungen**

Might this one be better?

Reply:
`Datenschutzbestimmungen` is used when referring to the body of law concerning data protection.

Looking at the top 20 German websites, it seems `Datenschutzerklärung` (data protection declaration) is most common, followed by simply `Datenschutz` (data protection). Looking at more official websites (such as the [German federal parliament](https://www.bundestag.de/) both are used as well.